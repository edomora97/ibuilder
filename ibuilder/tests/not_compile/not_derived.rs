struct Foo {
    field: i64,
}

fn main() {
    Foo::builder();
}
