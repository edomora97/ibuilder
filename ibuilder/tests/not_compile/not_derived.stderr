error[E0599]: no function or associated item named `builder` found for struct `Foo` in the current scope
 --> $DIR/not_derived.rs:6:10
  |
1 | struct Foo {
  | ---------- function or associated item `builder` not found for this
...
6 |     Foo::builder();
  |          ^^^^^^^ function or associated item not found in `Foo`
